cpp/include/cudf/detail/unary.hpp

```diff
@@ -74,6 +74,16 @@ std::unique_ptr<cudf::column> unary_operation(
   rmm::cuda_stream_view stream,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @copydoc cudf::is_valid
+ *
+ * @param stream CUDA stream used for device memory operations and kernel launches.
+ */
+std::unique_ptr<cudf::column> is_valid(
+  cudf::column_view const& input,
+  rmm::cuda_stream_view stream,
+  rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
```

Comment:
Since this is a new API being introduced by this PR, I think we can drop the defaults going by @vyasr's comment above, right?

Reply:
This is not new. It is just brought from the non-detail header. I want to keep consistency between it with all other detail APIs in the same file (which still have default value for `mr`).

Reply:
That's fine I can do batch removals later.